drivers/overlay/overlay.go

```diff
@@ -1719,26 +1721,29 @@ func (d *Driver) Exists(id string) bool {
 	return err == nil
 }
 
-// List layers (not including additional image stores)
+// ListLayers(), when implemented, is used when checking for errors.
```

Comment:
AFAICS it is actually used for `Wipe` (only?); this comment, and similar comments in other graph drivers, seem to be a bit misleading.

Reply:
I’m fine with keeping it as is, then.

(Or, alternatively, just document the driver’s responsibility, and let callers use it for any purpose they can think of. That’s what the `ProtoDriver`’s interface documentation does already.)